src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -143,43 +145,109 @@ class PeerAPI(override val ipManager: IPManager[IO])(
 
   private[p2p] def postEndpoints(socketAddress: InetSocketAddress) =
     post {
-      pathPrefix("snapshot") {
-        path("info") {
-          APIDirective.extractIP(socketAddress) { ip =>
-            entity(as[Array[Array[Byte]]]) { curCheckpointHashes =>
+      pathPrefix("snapshot" / "info") {
+        pathEnd {
+          entity(as[(String, Array[Array[Byte]])]) {
```

Comment:
This tuple was just from the due to issues using ip as uuid for 
dao.snapshotService.recentSnapshotInfo.put(ip, infoSer).put we can revert